tests/rados/test_osd_rebalance.py

```diff
@@ -0,0 +1,163 @@
+import datetime
```

Comment:
docstring missing about the intent of the module.

Reply:
added doc string